src/ripple/overlay/impl/Tuning.h

```diff
@@ -52,13 +52,13 @@ enum
 
     /** How many milliseconds to consider high latency
         on a peer connection */
-    peerHighLatency     =  250,
+    peerHighLatency     =  300,
 
     /** How often we check connections (seconds) */
-    checkSeconds        =   10,
+    checkSeconds        =   32,
 
     /** How often we latency/sendq probe connections */
-    timerSeconds        =    4,
+    timerSeconds        =    8,
```

Comment:
Bad time to change `checkSeconds` and `timerSeconds` into `std::chrono::seconds`?